config/iam/recommended-policy-arn

```diff
@@ -0,0 +1 @@
+AWSCloudTrail_FullAccess
```

Comment:
this should be an ARN :) Like https://github.com/aws-controllers-k8s/s3-controller/blob/main/config/iam/recommended-policy-arn ...

Reply:
Haaa.. fixed